admin_manual/installation/deployment_recommendations.rst

```diff
@@ -26,6 +26,15 @@ General Recommendations
 .. note:: Whatever the size of your organization, always keep one thing in mind: 
    *The amount of data stored in ownCloud will only grow. So plan ahead.*
 
+Do Not Use a Shared Host
```

Comment:
I wonder if this is clear enough what a "Shared Host" is or if there could be confusion there.

Reply:
e.g. I have a cloud instance with ownCube just on a basic personal plan. I have no command line access. I press the webUI button to upgrade every now and then. If it goes wrong (or usually to gives me some warning message about something and refuses to proceed), then I have to email their support. Is that a "Shared Host"?

Reply:
I think internally we agreed on using a wording like "we do not support hosters where no command line access nor cron is available".

ownCube doesn't fall into that category because they already host ownCloud themselves. Their admins have access to CLI and they can setup cron, so it's fine.

This is mostly about shared hosters that only provide you with a home folder with some PHP space and you only have FTP access to upload some files there and nothing more.

Reply:
Fair points. I'll make the wording more specific.